### PR TITLE
It was a hard project but yet was very fun to learn new stuff!

### PR DESCRIPTION
**Pull request by AhmadJer99**
I had fun learning sqlite3 and how to use it with C# also i tried to tidy up my code a little bit after reading an article about KISS and how i could apply  it, and am sorry if the code looks tangled inside i tried my best on this one .
anyways be  harsh with the review as usual.

### CODE REVIEW

File: HabitTracker/HabitTracker/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "HabitTracker": {
+      "commandName": "Project",
+      "workingDirectory": "C:\\Users\\AhmadJerjees\\source\\repos\\HabitTracker\\HabitTracker"
```
Comment (chrisjamiecarter) [inline HabitTracker/HabitTracker/Properties/launchSettings.json:5]: Remember that this hard-coded path will not exist on other peoples machines, you will need to include some info in README on changing this as a prerequisite step

File: HabitTracker/HabitTracker/habitlogger.db
Comment (chrisjamiecarter) [inline HabitTracker/HabitTracker/habitlogger.db:1]: Databases are not really supposed to exist in source control

**Review by chrisjamiecarter — APPROVED** [review]
Hey @AhmadJer99  👋,

Excellent work on your HabitTracker project submission! 🎉 

I have performed a peer review and have some praise, as well as a few suggestions:

🟢 **README File**
Well done on your readme file. It contains the right content, and was a perfect introduction to your application 🥇 .

🟢 **Separation of Concerns**
Good job on separating the concerns in your project, but grouping the controllers and the habit tracker related code and classes 🥈 .

🟢 **Parameterised SQL Queries**
Well done on parameterising your sql queries to combat sql injection attacks in your application, this will come in useful 🥉 .

🟢 **User Defined Habits**
Good job on implementing this challenge requirement ⭐ .

🟢 **Seed Data**
Seeding some data into the database make testing this application easier, so thank you for that 🌟 

🟢 **Report Functionality**
Fantastic implementation of this challenge requirement 🏅 

🟢 **User Experience**
Clean main menus and not cluttering of the console window 🎖️ 

🟠 **Solution Structure**
As per the [Code Reviews](https://thecsharpacademy.com/article/52/code-reviews) academy page, your project should have been submitted inside a `HabitTracker.AhmadJer99` directory (`[ProjectName].[GitHubUsername]`).
This will make it easier for code-reviewers to pull down and review your code in the future.

🟠 **Style Guide Compliance**
You have a lot of styling issues in your project.
Remember that when you have finished a project, give it a polish pass. Pay attention to any code conventions in place, in this instance refer to the C# Academy's [code-conventions](https://thecsharpacademy.com/article/58/code-conventions) page.
This is important as good code conventions/styling ensures consistency, makes your code easier for others to read and maintain, and improves your chances of getting a pull request reviewed and approved.

***

Feel free to reach out if you have any questions or if you'd like to collaborate further on any of these points 🆘 .

You're doing a fantastic job, and these are just a few suggestions to support your growth as a developer.

I will go ahead and mark as approved, keep up the excellent work on the next projects! 😊

Best regards,
@chrisjamiecarter 👍

**Comment by AhmadJer99** [comment]
Thanks for the review , i will make sure to look into the issues and re factor the code accordingly.

File: HabitTracker/HabitTracker/Properties/launchSettings.json (hunk 1 above)
Comment (AhmadJer99) [reply HabitTracker/HabitTracker/Properties/launchSettings.json:5]: oh i never gave it a thought to make a dynamic path that would work on every machine, apologies Mr , i will make sure to fix it.

**Pull request closed without merging**